web/src/pages.rs

```diff
@@ -160,9 +160,18 @@ pub async fn my_categories(
 ) -> WebResult<Response> {
     info!("Fetching categories for user_id: {}", user.user_id);
 
-    let categories = sqlx::query!("SELECT * FROM category",)
-        .fetch_all(&state.db)
-        .await?;
+    let categories = sqlx::query!(
```

Comment:
Noticed I was selecting all categories regardless of the `user_id`. So I changed it to make sure to selecting all `category_id` with the designated `user_id`

Reply:
Love this! Nice job on all the joins lol